js/letters.js

```diff
@@ -6,20 +6,55 @@ cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
 proident, sunt in culpa qui officia deserunt mollit anim id est laborum.`;
 
 var counter = {
-  "a" : 0,  "b" : 0,  "c" : 0,  "d" : 0,
-  "e" : 0,  "f" : 0,  "g" : 0,  "h" : 0,
-  "i" : 0,  "j" : 0,  "k" : 0,  "l" : 0,
-  "m" : 0,  "n" : 0,  "o" : 0,  "p" : 0,
-  "q" : 0,  "r" : 0,  "s" : 0,  "t" : 0,
-  "u" : 0,  "v" : 0,  "w" : 0,  "x" : 0,
-  "y" : 0,  "z" : 0
-}
+  a: 0,
+  b: 0,
+  c: 0,
+  d: 0,
+  e: 0,
+  f: 0,
+  g: 0,
+  h: 0,
+  i: 0,
+  j: 0,
+  k: 0,
+  l: 0,
+  m: 0,
+  n: 0,
+  o: 0,
+  p: 0,
+  q: 0,
+  r: 0,
+  s: 0,
+  t: 0,
+  u: 0,
+  v: 0,
+  w: 0,
+  x: 0,
+  y: 0,
+  z: 0
+};
 
-function countLetters(counter, sample_text){
-  // FIX ME
+function countLetters(counter, sample_text) {
+  if (sample_text.length === 1) {
+    counter[sample_text]++;
+    return;
+  }
+  let text = sample_text.toLowerCase();
+  let textArray = text.split("");
+  let currentChar = textArray.pop();
+  if (
+    currentChar !== " " &&
+    currentChar !== "." &&
+    currentChar !== "," &&
+    currentChar !== "\n"
+  ) {
+    counter[currentChar]++;
+  }
+  textString = textArray.join("");
```

Comment:
Instead of lower casing, then splitting, then popping off the last character in the string, then finally joining the array back into a string, see if you want just slice off the first or last character in the string passed into this function, then just lowercase that single character.